### PR TITLE
for taking the user to a workspace once the clone command is okay'ed and the backend returns with success status

### PR DESCRIPTION
**Pull request by eddiebroad**
for taking the user to a workspace once the clone command is okay'ed

### CODE REVIEW

**Comment by MattPutnam** [comment]
:+1: Seems to work for me.  Sending to @dmohs in case he has any comments about how the nav context is used.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs [hunk 1]
```diff
@@ -37,7 +38,8 @@
                    :dismiss-self #(swap! state dissoc :editing-acl?)
                    :update-owners #(swap! state update-in [:server-response :workspace] assoc "owners" %)}])
      (when (:cloning? @state)
-       [WorkspaceCloner {:dismiss #(swap! state dissoc :cloning?)
+       [WorkspaceCloner {:nav-context nav-context
+                         :dismiss #(swap! state dissoc :cloning?)
```
Comment (eddiebroad) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs]: I'll change :dismiss to :on-success and have it both clear the box from the screen and use the nav-context to navigate to the new clone workspace.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs [hunk 2]
```diff
@@ -37,7 +39,10 @@
                    :dismiss-self #(swap! state dissoc :editing-acl?)
                    :update-owners #(swap! state update-in [:server-response :workspace] assoc "owners" %)}])
      (when (:cloning? @state)
-       [WorkspaceCloner {:dismiss #(swap! state dissoc :cloning?)
+       [WorkspaceCloner {:on-success (fn [namespace name]
+                                       (utils/rlog (str "on-success called with namespace=" namespace " and name=" name   ))
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs]: No need for this log statement anymore.

**Comment by dmohs** [comment]
:+1: One errant log statement to remove. Merge when that's done. Looks great.

**Pull request merged**